engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java

```diff
@@ -35,6 +36,17 @@
     protected ChatMessageEvent() {
     }
 
+    /**
+     * A chat message is associated with an entity that sent it.
+     * <p>
+     * This entity may often be a player, but it can be any entity with a {@link org.terasology.logic.common.DisplayNameComponent}.
+     * If the entity also has a {@link org.terasology.network.ColorComponent} the sender's name will be colorized.
```

Comment:
```suggestion
     * If the entity also has a {@link org.terasology.network.ColorComponent}, the sender's name will be colorized.
```

Reply:
also not sure whether it's "colorized" or "coloured"

Reply:
🤷‍♂️ 🎨